cfpb/consumerfinance.gov
# Report SBL FIG search queries to Google Analytics

Log users' search terms to GA so we can see what folks search for on the Small Business Lending filing instructions guide page.

See https://github.local/reg-tech/sbl-data-collection/issues/308

## Changes

- Use our analytics module to send search terms to GA

## How to test this PR

1. I've been using https://github.com/sean-adams/dataslayer to confirm the datalayer is being updated.
1. After this PR is merged I'll ask the DA folks to check if the reporting is working.

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
